### ISSUE TITLE
Is it more difficult to train model?

### ISSUE THREAD

**Issue by yuanfuqiang456** [vacancy/Synchronized-BatchNorm-PyTorch#18]
when I use this sync-BN in my model , it became more slowly and even stop . I don not know why it happened,IO load? or other reasons?Can you help me?

**Comment by vacancy** [comment]
Hi there,
Thanks for your report. Can you provide more details on setup? How slow is it precisely?

**Comment by saurabhiit2007** [comment]
@vacancy I'm also facing the same issue. The code is getting stuck during the forward phase.

**Comment by jiangxiluning** [comment]
@saurabhiit2007 @vacancy For me it is the same issue, the first epoch is stuck after I get the first batch, seems like it has been blocked by some operations.

**Comment by vacancy** [comment]
Hi @jiangxiluning @saurabhiit2007 

Can this comment resolve the issue? https://github.com/vacancy/Synchronized-BatchNorm-PyTorch/issues/3#issuecomment-412139776
